layouts/partials/sidebar.html

```diff
@@ -11,17 +11,20 @@ <h1>
     <nav class="sidebar-nav">
       <ul class="current sidebar-main">
         {{ $currentPage := . }}
-        {{ range .Site.Menus.docs }}
+        {{ range site.Menus.docs }}
```

Comment:
Are you sure about this change? (I might be wrong since I dont have all the context, but it looks wrong to me)

Reply:
No, this is correct. Hugo now supports a `site` function that doesn't require you to perform gymnastics like `$.Site` or `$.Page.Site` or anything like that.